src/Storages/Distributed/DistributedSink.cpp

```diff
@@ -448,9 +448,9 @@ DistributedSink::runWritingJob(JobReplica & job, const Block & current_block, si
                     copy_query_ast,
                     job.local_context,
                     allow_materialized,
-                    /* no_squash */ false,
-                    /* no_destination */ false,
-                    /* async_isnert */ false);
+                    /* no_squash= */ false,
```

Comment:
Unimportant question: is there some guide or consensus saying which style we should use for these comments? Searching clickhouse code, I see occurrences of almost all possible styles: `/*foo*/false`, `/*foo*/ false`, `/* foo */ false`, `/* foo */false`, `/*foo=*/false`, `/* foo= */ false`, `/* foo = */ false`. Does something distinguish `/* foo= */ false` as the Schelling point for clickhouse codebase? Should I switch to it?

Reply:
Oh, this is a good question, this style is checked by the clang, so if the name in the comment will not match with the name of the parameter it will throw an error.

Reply:
One tiny note, it will not work obviously for i.e. `make_unique` and friends